readme.md

```diff
@@ -147,46 +162,78 @@ Once installation is working, try creating and requesting resources. Note that t
 
 * **Your Answer:**
+Add useFindAndModify: false to the mongoose.connect(MONGO_DB_CONNECTION, { useNewUrlParser: true }) line in the app.js file
+
+e.g. change it to:
+
+mongoose.connect(MONGO_DB_CONNECTION, { useNewUrlParser: true, useFindAndModify: false })
 
 ---
 
-- [ ] Update the API so that you can successfully delete a record. Use the [Model.findOneAndDelete()](https://mongoosejs.com/docs/api.html#model_Model.findOneAndDelete) method.
+- [x] Update the API so that you can successfully delete a record. Use the [Model.findOneAndDelete()](https://mongoosejs.com/docs/api.html#model_Model.findOneAndDelete) method.
 
 ---
 
-- [ ] A lot of information is being returned on each request. Some of this information we may want to keep private. Using the [.select()](https://mongoosejs.com/docs/api.html#query_Query-select) method, return only the "_id", "title", "start_year", and "season_count" on all the above requests.
+- [x] A lot of information is being returned on each request. Some of this information we may want to keep private. Using the [.select()](https://mongoosejs.com/docs/api.html#query_Query-select) method, return only the "_id", "title", "start_year", and "season_count" on all the above requests.
 
 * **Question:** At least one of these will throw an error. Which one? How can you get around this issue?
 
 * **Your Answer:** 
+I didn't get an error for any of them!
+I am using:
+
+``` js
+const response = await Series.find().select('_id title season_count start_year')
+```
+
+and it returns that data without error. What error should I be getting?
 
 ---
 
-- [ ] Modify your `GET /api/series` route so that you can search through the content based off of query parameters. For example, if your request was `GET /api/series?start_year=1997`, it would return all series that start in 1997.
+- [x] Modify your `GET /api/series` route so that you can search through the content based off of query parameters. For example, if your request was `GET /api/series?start_year=1997`, it would return all series that start in 1997.
 
 ---
 
-- [ ] At the moment, there is no validation set on creating any series. Add validation so that each of the three fields are required in order to post to series. Handle any errors that arise with a status code 400 and a brief, standardized message.
+- [x] At the moment, there is no validation set on creating any series. Add validation so that each of the three fields are required in order to post to series. Handle any errors that arise with a status code 400 and a brief, standardized message.
 
 * **Question:** You may notice that you can add custom error messages to individual fields failing. Try adding these and take a look at the error message received. How can you make use of those specific messages for each field?
 
 * **Your Answer:**
 
+You can make use of them by providing useful errors to the user so they can correct their input mistakes. 
```

Comment:
Correct. You can access these custom messages by using `try/catch` or `.catch()` and then digging into the provided error.